KosherCocoa/Library/Core/Solar/KCAstronomicalCalculator.h

```diff
@@ -14,7 +14,7 @@
  *  A protocol defining methods to return sunrise and sunset times.
  */
 NS_SWIFT_NAME(AstronomicalCalculator)
-@protocol KCAstronomicalCalculator  <NSObject>
+@interface KCAstronomicalCalculator  <NSObject>
```

Comment:
We probably want to change `<NSObject>`  to `: NSObject`. (From conformance to subclassing.)